### PR TITLE
Fix for issue #2663

### PR DESCRIPTION
**Pull request by ghost**
Corrected web.metrics.prometheus.buckets to output a slice of floats to configmap

Corrected web.metrics.datadog.address to output a string to configmap

Corrected web.metrics.statsd.address to output a string to configmap

Updated values.yaml web.metrics.datadog.pushInterval to match documentation

Updated values.yaml web.metrics.statsd.pushInterval to match documentation

Removed extraneous whitespace from web.metrics section of the configmap template

Updated README.md to reflect changes

### CODE REVIEW

**Comment by TimDumol** [comment]
Hi @troywilson , your PR has merge conflicts that need to be resolved. And thanks for working for this, I experienced this myself!

**Comment by unguiculus** [comment]
Could you fix conflicts, please.

File: stable/traefik/templates/configmap.yaml [hunk 1]
```diff
@@ -88,17 +88,32 @@ data:
...
-    {{ if .Values.metrics.datadog.enabled }}
+    {{- if .Values.metrics.datadog.enabled }}
     [web.metrics.datadog]
-      Address = {{ .Values.metrics.datadog.address }}
-      PushInterval = "{{ .Values.metrics.datadog.pushInterval }}"
+      {{- if .Values.metrics.datadog.address }}
+      address = {{ .Values.metrics.datadog.address | quote }}
+      {{- end}}
+      {{- if .Values.metrics.datadog.pushInterval }}
+      pushInterval = {{ .Values.metrics.datadog.pushInterval | quote }}
```
Comment (unguiculus) [inline stable/traefik/templates/configmap.yaml]: Just checked the docs. `pushinterval` is all lower case. 

https://docs.traefik.io/configuration/backends/web/#statsd

It looks like this is pretty inconsistent. Some start with a lower case letter, some with an upper case letter, some are camel case, some all lower case. Oh my...

**Review by unguiculus — CHANGES_REQUESTED** [review]

File: stable/traefik/templates/configmap.yaml (hunk 1 above)
Comment (ghost) [reply stable/traefik/templates/configmap.yaml]: pushinterval is now lowercase, in line with all the other variables.

The docs are inconsistent, Traffic doesn't seem seem to care about the case of the config variables...yet

**Review by unguiculus — APPROVED** [review]

**Pull request merged**